ts/store/transforms/dateISO8601Tranform.ts

```diff
@@ -42,14 +46,18 @@ const dateReviver = (key: any, value: any): any => {
 };
 
 const encoder: TransformIn<any, string> = (value: any, _: string): any =>
-  JSON.parse(JSON.stringify(value), dataReplacer);
+  value !== undefined
```

Comment:
```suggestion
  fromNullable(value).fold(undefined, v =>
    JSON.parse(JSON.stringify(v), dataReplacer)
  );
```
It's pretty the same but using _Option_